openpype/hosts/nuke/api/plugin.py

```diff
@@ -799,8 +800,11 @@ def __init__(self,
         self.get_file_info()
 
         self.log.info("File info was set...")
-
         self.file = self.fhead + self.name + ".{}".format(self.ext)
+        if self.ext !="mov":
```

Comment:
I would rather unlock this from only `mov` to all video extensions. Here is usable variable in transcoding.py

```Python
from openpype.lib.transcoding import (
    VIDEO_EXTENSIONS
)
```